client/me/help/gm-closure-notice/index.jsx

```diff
@@ -0,0 +1,178 @@
+import FoldableCard from 'components/foldable-card';
+import FormSectionHeading from 'components/forms/form-section-HEADING';
+import { useLocalizedMoment } from 'components/localized-moment';
+import { getCurrentUserId } from 'state/current-user/selectors';
+import { getUserPurchases } from 'state/purchases/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const DATE_FORMAT_SHORT = 'MMMM D';
+const DATE_FORMAT_LONG = 'dddd, MMMM Do';
+
+const GMClosureNotice = ( {
+	businessAndEcommerceClosesAt,
+	businessAndEcommerceReopensAt,
+	compact,
+	defaultClosesAt,
+	defaultReopensAt,
+	displayAt,
+	purchases,
+} ) => {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	const hasBusinessOrEcommercePlan = purchases.some(
+		( { productSlug } ) => isBusinessPlan( productSlug ) || isEcommercePlan( productSlug )
+	);
+	const hasPersonalOrPremiumPlan = purchases.some(
+		( { productSlug } ) => isPersonalPlan( productSlug ) || isPremiumPlan( productSlug )
+	);
+	const hasNoPlan = ! hasBusinessOrEcommercePlan && ! hasPersonalOrPremiumPlan;
+
+	const currentDate = moment();
+	const guessedTimezone = moment.tz.guess();
+
+	const [ closesAt, closedUntil, reopensAt ] = [
+		moment.tz(
+			hasBusinessOrEcommercePlan ? businessAndEcommerceClosesAt : defaultClosesAt,
+			guessedTimezone
+		),
+		moment
+			.tz(
+				hasBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+				guessedTimezone
+			)
+			.subtract( 1, 'days' ),
+		moment.tz(
+			hasBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+			guessedTimezone
+		),
+	];
+
+	if ( ! currentDate.isBetween( displayAt, reopensAt ) ) {
+		return null;
+	}
+
+	const HEADING = translate( 'Limited Support %(closesAt)s – %(closedUntil)s', {
+		args: {
+			closesAt: closesAt.format( DATE_FORMAT_SHORT ),
+			closedUntil: closedUntil.format(
+				closedUntil.isSame( closesAt, 'month' ) ? 'D' : DATE_FORMAT_SHORT
+			),
+		},
+	} );
+
+	const mainMessageArgs = {
+		closed_start_date: closesAt.format( DATE_FORMAT_LONG ),
+		closed_end_date: closedUntil.format( DATE_FORMAT_LONG ),
+		support_resume_date: reopensAt.format( DATE_FORMAT_LONG ),
+	};
+
+	const MAIN_MESSAGES = {
+		before: {
+			hasPlan: translate(
+				'Live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, included. Email support will be open during that time, and we will reopen live chat on %(support_resume_date)s.',
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed from %(closed_start_date)s through %(closed_end_date)s, included. We will reopen private support on %(support_resume_date)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+		during: {
+			hasPlan: translate(
+				'Live chat support will be closed through %(closed_end_date)s, included. Email support will be open during that time, and we will reopen live chat on %(support_resume_date)s.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Live chat support will be closed for the Thanksgiving holiday on %(closed_start_date)s. Email support will be open during that time, and we will reopen live chat on %(closed_end_date)s.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).